src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/Reaction/AngularDistribution/src/ACEify.hpp

```diff
@@ -0,0 +1,27 @@
+template< typename = int >
+void ACEify( Table::Data& tData, double& LOCB ) {
+
+  auto& xss = tData.XSS();
+  auto jxsRelative = tData.JXS( 9 );
+
+  LOCB = ( xss.size() - jxsRelative + 2 );
+  
+  auto& grid = this->energyGrid;
+
+  xss.push_back( grid.size() );
+  xss |= ranges::action::push_back( this->energyGrid );
+  auto LC_i = xss.size() + 1;
+  xss |= ranges::action::push_back( ranges::view::repeat_n( 0, grid.size() ) );
+
+  auto enumerated = ranges::view::enumerate( this->representations );
+  for( auto it = enumerated.begin(); it != enumerated.end(); ++it ){
+    decltype( auto ) index = std::get< 0 >( *it );
```

Comment:
this doesn't appear to be used

Reply:
If you're really not using this, no need to use the view::enumerate. You can loop over the actual representations vector directly.

Reply:
yep, I've changed how I did it since the `for` loop was first constructed. Nice find!